src/Umbraco.Cms.Api.Management/Controllers/MediaType/Item/AllowedMediaTypeItemController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Asp.Versioning;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Api.Management.ViewModels.MediaType.Item;
+using Umbraco.Cms.Core.Mapping;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Services.ContentTypeEditing;
+
+namespace Umbraco.Cms.Api.Management.Controllers.MediaType.Item;
+
+[ApiVersion("1.0")]
+public class AllowedMediaTypeItemController : MediaTypeItemControllerBase
+{
+    private readonly IMediaTypeEditingService _mediaTypeEditingService;
+    private readonly IUmbracoMapper _mapper;
+
+    public AllowedMediaTypeItemController(IMediaTypeEditingService mediaTypeEditingService, IUmbracoMapper mapper)
+    {
+        _mediaTypeEditingService = mediaTypeEditingService;
+        _mapper = mapper;
+    }
+
+    [HttpGet("allowed")]
+    [MapToApiVersion("1.0")]
+    [ProducesResponseType(typeof(PagedModel<MediaTypeItemResponseModel>), StatusCodes.Status200OK)]
+    public async Task<IActionResult> Item(CancellationToken cancellationToken, string fileExtension, int skip = 0, int take = 100)
+    {
+        IMediaType[] mediaTypes = (await _mediaTypeEditingService.GetMediaTypesForFileExtension(fileExtension)).ToArray();
```

Comment:
now this is a new method, I think we should handle the skip/take in the service layer